OutOfSchool/OutOfSchool.WebApi/Services/Images/ImageService.cs

```diff
@@ -106,7 +106,7 @@ public async Task<Result<ImageDto>> GetByIdAsync(string imageId)
                     var validationResult = validator.Validate(stream);
                     if (!validationResult.Succeeded)
                     {
-                        logger.LogError($"Image with {nameof(fileCollection)} id = {i} isn't valid.");
+                        logger.LogError($"Image with {nameof(fileCollection)} id = {i} isn't valid: {string.Join(",", validationResult.Errors)}");
```

Comment:
we will return only the first error if not succeeded in this case

Reply:
join is more general then `array[0]` :)